performance-test/conf/README.md

```diff
@@ -0,0 +1,20 @@
+# What is this for?
+
+This folder should be used to store configuration files used by Kedro or by separate tools.
```

Comment:
Is there any specific configuration needed to be documented? Otherwise I think we can remove this from our project